src/engine.c

```diff
@@ -34,13 +34,48 @@ char *exp_names[MAX_EXPANSION + 1] =
 	"The Brink of War",
 	"Alien Artifacts",
 	"Xeno Invasion",
+	"Rebel vs Imperium (without Gathering Storm)",
```

Comment:
Could you shorten this somewhat to avoid cluttering the UI? Suggestions: "Rebel vs Imperium (only)", "Rebel vs Imperium (no GS)"